docs/integrations/editors.md

```diff
@@ -148,6 +187,8 @@ curl https://raw.githubusercontent.com/psf/black/stable/autoload/black.vim -o ~/
 Let me know if this requires any changes to work with Vim 8's builtin `packadd`, or
 Pathogen, and so on.
 
+#### Usage
+
 This plugin **requires Vim 7.0+ built with Python 3.7+ support**. It needs Python 3.7 to
```

Comment:
This paragraph doesn't really fit under "Usage". Maybe put it at the top of the "Installation" section?

Reply:
Yep good call, moved.